### PR TITLE
[LWG 10] P2675R1 format's width estimation is too approximate and not forward …

### LINKED ISSUE #6100: [2023-02 LWG Motion 10]  P2675R1 format's width estimation is too approximate and not forward compatible

### PR DESCRIPTION
**Pull request by burblebee**
Fixes NB FR 7-012 (C++23 CD).

Fixes #6100.
Fixes cplusplus/papers#1341
Fixes cplusplus/nbballot#409

### CODE REVIEW

File: source/utilities.tex [hunk 1]
```diff
@@ -14997,20 +14997,16 @@
...
-\item \ucode{fe30} -- \ucode{fe6f}
-\item \ucode{ff00} -- \ucode{ff60}
-\item \ucode{ffe0} -- \ucode{ffe6}
-\item \ucode{1f300} -- \ucode{1f64f}
-\item \ucode{1f900} -- \ucode{1f9ff}
-\item \ucode{20000} -- \ucode{2fffd}
-\item \ucode{30000} -- \ucode{3fffd}
+\item
+Any code point with the East_Asian_Width="W" or
+East_Asian_Width="F" Derived Extracted Property as described by
```
Comment (JohelEGP) [inline source/utilities.tex:15002]: I think these should be formatted with code font.
```
$ git grep -C 1 'Unicode property' *.tex
utilities.tex-\placeholder{C} corresponds to either
utilities.tex:a UCS scalar value whose Unicode property \tcode{General_Category}
utilities.tex-has a value in the groups \tcode{Separator} (\tcode{Z}) or \tcode{Other} (\tcode{C}) or to
utilities.tex:a UCS scalar value which has the Unicode property \tcode{Grapheme_Extend=Yes},
utilities.tex-as described by table 12 of \UAX{44}, or
```

File: source/utilities.tex [hunk 2]
```diff
@@ -14997,20 +14997,16 @@
...
+\UAX{44}
+\item
+\ucode{4dc0} - \ucode{4dff} (Yijing Hexagram Symbols)
+\item
+\ucode{1f300} - \ucode{1f5ff} (Miscellaneous Symbols and Pictographs)
+\item
+\ucode{1f900} - \ucode{1f9ff} (Supplemental Symbols and Pictographs)
```
Comment (JohelEGP) [inline source/utilities.tex:15009]:
```suggestion
\item
\ucode{4dc0} -- \ucode{4dff} (Yijing Hexagram Symbols)
\item
\ucode{1f300} -- \ucode{1f5ff} (Miscellaneous Symbols and Pictographs)
\item
\ucode{1f900} -- \ucode{1f9ff} (Supplemental Symbols and Pictographs)
```

**Review by JohelEGP — CHANGES_REQUESTED** [review]

File: source/utilities.tex (hunk 1 above)
Comment (burblebee) [reply source/utilities.tex:15002]: I \tcode'd the `East_Asian_Width` assignments, but what about "Derived Extracted Property"?

Comment (JohelEGP) [reply source/utilities.tex]: Seems fine as is.

**Pull request merged**